nautobot/extras/plugins/views.py

```diff
@@ -198,3 +200,24 @@ def get(self, request, format=None):  # pylint: disable=redefined-builtin
                 )
             )
         )
+
+
+class MarketplaceView(GenericView):
+    """
+    View for listing all available Apps.
+    """
+
+    def get(self, request):
+        file_path = os.path.dirname(os.path.abspath(__file__)) + "/marketplace_manifest.yml"
+        with open(file_path, "r") as yamlfile:
+            marketplace_data = yaml.safe_load(yamlfile)
+
+        installed_apps = [app for app in apps.get_app_configs() if app.name in settings.PLUGINS]
+
+        visible_marketplace_apps = [
+            app
+            for app in marketplace_data["apps"]
+            if all(installed_app.name != app["package_name"] for installed_app in installed_apps)
+        ]
```

Comment:
This is hiding apps that are already installed, right? Might be nice to have a comment to that effect.

Reply:
852b3ffb3b9a95d1c402943f232dd21565b5bfd4